CMakeLists.txt

```
@@ -85,6 +85,15 @@ include_directories(
   ${CMAKE_CURRENT_SOURCE_DIR}/include
   ${CMAKE_CURRENT_BINARY_DIR}/include)
 
+############## Code Coverage #########################
+option(ENABLE_COVERAGE "Enable coverage reporting." OFF)
+
+if (ENABLE_COVERAGE)
+  include(CodeCoverage)
+  APPEND_COVERAGE_COMPILER_FLAGS()
+endif()
+
+
```

```suggestion
add_feature_info("Coverage" ENABLE_COVERAGE "analyze test coverage and generate report")
```

Inserting this line (along with a merge of #383) will add a Coverage entry to the FeatureSummary output at the end of the configuration run.